### ISSUE TITLE
para-cli throw exception

### ISSUE THREAD

**Issue by mzch** [Erudika/para-cli#2]
when executing para-cli@**1.9.3** outside /usr/lib/node_modules/para-cli, it throw an exception.

```
/usr/lib/node_modules/para-cli/node_modules/conf/index.js:58
				throw new Error('Project name could not be inferred. Please specify the `projectName` option.');
				^

Error: Project name could not be inferred. Please specify the `projectName` option.
    at new Conf (/usr/lib/node_modules/para-cli/node_modules/conf/index.js:58:11)
    at Object.<anonymous> (/usr/lib/node_modules/para-cli/para-cli.js:88:14)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)
    at internal/main/run_main_module.js:17:11
```

**Comment by JinghengHuang** [comment]
Same execption here, on a Windows system.

**Comment by albogdano** [comment]
Fixed in `1.9.4` :+1:
